hw/bsp/stm32h5/family.c

```diff
@@ -59,23 +59,10 @@ UART_HandleTypeDef UartHandle;
 
 void board_init(void) {
   HAL_Init(); // required for HAL_RCC_Osc TODO check with freeRTOS
-  SystemClock_Config(); // implemented in board.h
+  board_clock_init(); // implemented in board.h
```

Comment:
please keep the function name SystemClock_Config() as it is

Reply:
Ok, I will restore original name.